content/sensu-go/5.5/reference/tessen.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Tessen"
+description: "Tessens sends anonymized data about the Sensu instance to Sensu Inc. You can use sensuctl to view and manage Tessen configuration. Read the reference to configure Tessen."
+weight: 10
+version: "5.5"
+product: "Sensu Go"
+menu: 
+  sensu-go-5.5:
+    parent: reference
+---
+
+- [Configuring Tessen](#configuring-tessen)
+- [Specification](#tessen-specification)
+- [Example](#tessen-example)
+
+## What is Tessen?
+
+Tessen is the Sensu call-home service.
+Included with every Sensu backend, Tessen sends anonymized data about the Sensu instance to Sensu Inc. every six hours, including:
+
+- The Sensu version
+- The number of Sensu agents and backends in the cluster
+- Whether the Sensu instance uses an [enterprise license][4]
+
+All data submissions are logged for complete transparency and transmitted over HTTPS.
+[Read the blog post][1] to learn more about Tessen.
+
+## Configuring Tessen
+
+You can use the [Tessen API][2] and [sensuctl][3] to view and manage Tessen configuration.
+Tessen is enabled by default on all Sensu backends.
+To manage Tessen configuration using sensuctl, configure sensuctl as the default [`admin` user](../../reference/rbac#default-user).
+[Licensed][4] Sensu instances cannot opt out of Tessen.
```

Comment:
Theoretically, they can opt out, however their configuration option will be overridden with opt in at runtime. I'm not sure if that's worth noting.